src/utils/line.ts

```diff
@@ -340,18 +342,19 @@ export const formatLinePageData = (
 
   positions?.map((position, i) => {
     console.log('positions info origin', positions);
-    console.log('position', position);
-
+    console.log('position', position.status);
+    //@ts-ignore
+    const status = position.status === 'OPEN' ? 'OPENED' : position.status;
     let Frate = normalizeAmount(position.fRate, 2);
     let Drate = normalizeAmount(position.dRate, 2);
-
+    console.log('in line.ts', position.status);
     let positionObject = {
-      status: position.status,
-      drate: Drate,
-      frate: Frate,
+      status,
+      dRate: Drate,
+      fRate: Frate,
```

Comment:
variable names shouldn't be Capitalized.

Reply:
@Seroxdesign tagging you here as this `formatGetBorrowerQuery` is your function from your PR.

Reply:
Got it will fix that